apps/OpenSign/public/locales/fr/translation.json

```diff
@@ -0,0 +1,591 @@
+     "quick-send-alert-2":"Veuillez vous assurer qu'au moins un widget de signature est ajouté pour tous les destinataires.",
+     "quick-send-alert-3":"Veuillez ajouter au moins un rôle à ce modèle afin d'en « envoyer rapidement » des copies à plusieurs signataires.",
+     "quick-send-alert-4":"L'envoi rapide a atteint la limite.",
+     "copy-link" :"Copier le lien",
+     "copy":"Copier",
+     "revoke-document":"Révoquer le document",
+     "revoke-document-alert" :"Êtes-vous sûr de vouloir révoquer ce document ?",
+     "resend-mail":"Renvoyer le courrier",
+     "resend-mail-help" :"Vous pouvez utiliser les variables suivantes qui seront remplacées par leurs valeurs réelles : - {{document_title}}, {{sender_name}}, {{sender_mail}}, {{sender_phone}}, {{receiver_name}}, {{receiver_email} }, {{receiver_phone}}, {{expiry_date}}, {{company_name}}, {{signing_url}}.",
+     "subject":"Sujet",
+     "body":"Corps",
+     "add-contact":"Ajouter le contact",
+     "add-signer-alert":"Le contact existe déjà !",
+     "record-delete-alert":"Enregistrement supprimé avec succès !",
+     "record-revoke-alert":" Enregistrement révoqué avec succès ! ",
+     "mail-sent-alert":"Courriel envoyé avec succès.",
+     "document-sent-alert":"Document envoyé avec succès !",
+     "template-share-alert":"Modèle partagé avec succès.",
+     "template-public-alert-1":"Vous avez rendu le modèle public avec succès.",
+     "template-public-alert-2":"Vous avez réussi à rendre le modèle privé.",
+     "template-public-alert-3":"Vous devez sélectionner un rôle pour les signataires publics.",
+     "template-public-alert-4" :"Veuillez vous assurer qu'au moins un widget de signature est ajouté pour tous les signataires.",
+     "template-public-alert-5" :"Étant donné que l'envoi dans l'ordre est activé sur ce modèle, le rôle public doit être positionné en haut.",
+     "template-public-alert-6" :"Veuillez associer des signataires à tous les rôles, à l'exception du rôle d'utilisateur public.",
+     "template-public-alert-7" :"Veuillez attribuer au moins un rôle public pour rendre ce modèle public.",
+     "loader":"Cela pourrait prendre un certain temps",
+     "pages":"Pages",
+     "document-signed":"Document signé",
+     "close":"Fermer",
+     "validation-alert":"Alerte de validation",
+     "validate-alert-mssg" :"L'entrée ne répond pas aux critères définis par l'expression régulière.",
+     "otp-verification":"Vérification OTP",
+     "enter-otp":"Entrez OTP",
+     "verify":"Vérifier",
+     "resend":"Renvoyer",
+     "verify-email":"Veuillez vérifier votre email!",
+     "send-otp":"envoyer un code à usage unique",
+     "otp-placeholder":"Saisissez l'OTP reçu par e-mail",
+     "loading-doc":"Chargement du document..",
+     "widgets-name":{
+        "signature":"signature",
+        "stamp":"estampiller",
+        "initials":"initiales",
+        "name":"nom",
```

Comment:
"name":"Nom et Prénom",

User can missunderstand and only write his Name and not firstname